src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -396,6 +397,10 @@ export const Details: React.FC<{
         )
     }
 
+      const showApplicationDetailedModal = (): void => {
+        toggleDetailedStatus && toggleDetailedStatus(true)
```

Comment:
Why do we have to put a check for `toggleDetailedStatus`, it's the component state setter, right?